### PR TITLE
test(dataproxy): add query logging tests

### PR DESCRIPTION
**Pull request by aqrln**
Add query logging tests for PostgreSQL and MongoDB with Data Proxy.

Closes https://github.com/prisma/client-planning/issues/292

### CODE REVIEW

**Review by janpio — COMMENTED** [review]
Nice.

File: dataproxy/nodejs-mongodb-logs/test.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -eu
+
+yarn test
```
Comment (aqrln) [inline dataproxy/nodejs-mongodb-logs/test.sh]:
```suggestion
pnpm test
```

File: dataproxy/nodejs-postgresql-logs/test.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -eu
+
+yarn test
```
Comment (aqrln) [inline dataproxy/nodejs-postgresql-logs/test.sh]:
```suggestion
pnpm test
```

**Comment by Jolg42** [comment]
Currently failing with ` MISSING DEP  Can not find dependency 'vite'` https://github.com/prisma/ecosystem-tests/actions/runs/4884925303/jobs/8718386856?pr=3456#step:8:547

**Review by Jolg42 — APPROVED** [review]

**Pull request merged**